app/models/notification.rb

```diff
@@ -79,7 +83,22 @@ def unarchive_if_updated
   def update_from_api_response(api_response, unarchive: false)
     attrs = Notification.attributes_from_api_response(api_response)
     self.attributes = attrs
+    set_author(api_response)
     unarchive_if_updated if unarchive
     save(touch: false) if changed?
   end
+
+  private
+
+  def set_author(api_response)
+    /\/((?:issues|pulls)\/(?<issue_number>\d+))|((?:commits)\/(?<commit_sha>[0-9a-f]{5,40}))\z/ =~ api_response[:subject][:url]
+    repo_name = api_response[:repository][:full_name]
+    self.subject_author = if issue_number
+                            issue = user.github_client.issue(repo_name, issue_number)
```

Comment:
Gaaah thank you for that. I honestly forgot about it, I have 0 private repos I'm watching and I forget it all the time.

Ugh I don't really know what to do here I guess. @andrew it sounds like you're still too hesitant for octobox.io to ask for repo scope so I'm guessing I should just have this behind a feature toggle. 

I'm hesitant to do that though as it seems like we're sort of reaching a critical mass where to get any more extremely helpful features we just sort of need to bite the bullet and do it one of these times. There's just so much more info given when you can actually query issues themselves.